src/screens/volunteer/VolunteerIndexScreen.tsx

```diff
@@ -106,11 +109,11 @@ export const VolunteerIndexScreen = ({ navigation, route }: StackScreenProps<any
           ListEmptyComponent={
             showCalendar ? (
               <Calendar
+                isListRefreshing={isLoading}
                 query={query}
                 queryVariables={queryVariables}
-                calendarData={data}
-                isLoading={isLoading}
                 navigation={navigation}
+                additionalData={data}
               />
```

Comment:
I see there is no alphabetical sorting here either.

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/pull/940/commits/f653378eefcca79cef1def80dfa038e995b1bccf